doc/man1/x509.pod

```diff
@@ -121,6 +122,11 @@ to attempt to obtain a functional reference to the specified engine,
 thus initialising it if needed. The engine will then be set as the default
 for all available algorithms.
 
+=item B<-preserve_dates>
+
+When signing a certificate, preserve the "notBefore" and "notAfter" dates instead
+of adjusting them to current time and duration.
```

Comment:
This confused me a bit. Does this mean preserve the "notBefore" and "notAfter" dates from the CSR? Also, how does this option interacts with the "-days" option. What happens if you supply both?